pages/02.applications/04.wishlist/apps_wishlist.md

```diff
@@ -306,6 +314,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [upmpdcli](https://www.lesbonscomptes.com/upmpdcli/) |  | [Upstream](https://framagit.org/medoc92/upmpdcli) |  |
 | [Uwazi](https://www.uwazi.io/) |  | [Upstream](https://github.com/huridocs/uwazi) |  |
 | [Volumio](https://volumio.org) |  | [Upstream](https://github.com/volumio) |  |
+| [Vikunja](https://vikunja.io/) |  | [Upstream](https://kolaente.dev/vikunja/api) |  |
```

Comment:
Vikunja can also be removed 😅 -> https://github.com/YunoHost-Apps/vikunja_ynh

Reply:
wow, cool)

Reply:
The package as still some issues...